### ISSUE TITLE
How can we make it responsive?

### ISSUE THREAD

**Issue by gurumark** [mickaelr/jquery-timelineMe#7]
I am trying to use your plugin for displaying comments. For large comments, it shrinks the comment container. It would be better if it moved the time line from center to the left or right. Or do you have any other idea?

**Comment by mickaelr** [comment]
Yep that's exactly what I have in mind: for screens < 640px wide, move the line to the left.
For the moment I'm trying to make it usable in horizontal mode, but responsive is the following issue I'll work on

**Comment by kyan001** [comment]
tablets' width are 768px (such like ipad), so I suggest screens < 768px instead of < 640px.
